Themis/Views/Assessment/CodeEditor/CodeView.swift

```diff
@@ -73,13 +75,13 @@ class ViewController: UIViewController {
     }
 }
 
-struct CodeView_Previews: PreviewProvider {
+/*struct CodeView_Previews: PreviewProvider {
     static var file = CodeFile(id: UUID(),
                                title: "Baum.java",
                                code: "class Baum {\n\tString type = \"Eiche\";\n}")
     static var previews: some View {
-        CodeView(model: AssessmentViewModel.mock)
+        CodeView(vm: CodeEditorViewModel())
             .padding()
             .previewInterfaceOrientation(.landscapeLeft)
     }
-}
+}*/
```

Comment:
Like above, I would suggest to either try to make it work or remove it completely